src/api/order/validations/findOrder.js

```diff
@@ -0,0 +1,26 @@
+import Joi from 'joi';
```

Comment:
Completely wrong file/fileName

If this file is validations/findOrder, then why are you checking order status here in this file? We will have more APIs when user will get order details, the validation you wrote here is for validating only for changeStatus API, but the name you used for your file is findOrder. Validate only if order exists, and don't validate orderStatus here in findOrder validation